### ISSUE TITLE
create_model函数中硬编码的13是指什么？

### ISSUE THREAD

**Issue by nonva** [kyzhouhzau/BERT-NER#39]
logits = tf.reshape(logits, [-1, FLAGS.max_seq_length, 13])  不该是num_labels？ 这里13是指啥

**Comment by fengchi863** [comment]
应该改为num_lables，他的get_labels是13个

**Comment by chensvm** [comment]
那不应该是12么？get_labels里面一共12个label
